LBH.AdultSocialCare.Api/Startup.cs

```diff
@@ -356,7 +354,6 @@ public static void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             IMapper mapper = app.ApplicationServices.GetService<IMapper>();
             ApiToDomainFactory.Configure(mapper);
             DomainToEntityFactory.Configure(mapper);
-            EntityToDomainFactory.Configure(mapper);
```

Comment:
There is automapper in this file. Will this not cause a problem mapping entities to domains?

Reply:
Sorry yes - I had deleted the file, will revert this